composer.json.~1.12.0.dist

```diff
@@ -0,0 +1,87 @@
+{
```

Comment:
@rafalmal I don't think this will work. 
Have you tried installing this plugin on clean Sylius 1.12?
Sylius 1.12 requires only php 8.x (it wont work on php 7.x) and symfony ^5.4 or ^6.0

Reply:
Yes i tried it on sylius 1.12 with symfony 5.4, but in this case i will remove php 7.3 and 7.4 from dependencies,

Reply:
I think it should work on sylius 1.12 with symfony 6 also

Reply:
we have oauth-server package wich doesnt work with symfony 6

Reply:
It is not needed in sylius 1.12 anymore

Reply:
Please verify this, but if I remember correctly,  friendsofsymfony/oauth-server-bundle is not needed is sylius 1.11 and 1.10 also. You can remove it completely